tests/request-fragment.js

```diff
@@ -41,6 +42,34 @@ describe('requestFragment', () => {
         });
     });
 
+    it('Should request fragment using https protocol and gzip compression', (done) => {
+        nock('https://fragment')
+            .defaultReplyHeaders({
+                'content-encoding': 'gzip',
+            })
+            .get('/')
+            .reply(200, () => {
+                const text = 'HTTP+GZIP';
+                const buf = new Buffer(text, 'utf-8');
+                return zlib.gzipSync(buf);
+            });
+
+        requestFragment('https://fragment/', fragmentAttrb, {
+            headers: {}
+        }).then((response) => {
+            let data = '';
```

Comment:
I know this just a test but this is a wrong way to assemble chunks. Here's the right one:
https://github.com/zalando-incubator/perron/pull/6